src/civis/response.py

```diff
@@ -400,6 +401,29 @@ def __next__(self):
         return next(self._iter)
 
 
+class ListResponse(list):
```

Comment:
The class name `ListResponse` was chosen to match the pattern of the existing `PaginatedResponse`.